R/get-named-regions.R

```diff
@@ -45,6 +45,7 @@ wb_get_named_regions_tab <- function(wb) {
     #localSheetId is not always available
     name = wb$tables$tab_nam,
     value = "table",
+    # TODO Maybe remove this, and change it for another function?
```

Comment:
why do you want to remove this? anything bad with it? it is only used in `wb_to_df()` where it should work. Please remove the comment

Reply:
My mistake.